build2.py

```diff
@@ -232,8 +298,25 @@ def cli():
     build_parser.add_argument("-noautofail", action='store_false', dest='autofail', default=True)
     split_parser = subp.add_parser('split')
     split_parser.add_argument('-t','--targetnum', type=int, default=10, help="How many packages in one anaconda build submission typically.")
-    split_parser.add_argument('-s','--split-files',type=str)
-    args = p.parse_args()
+    split_parser.add_argument('-s','--split-files',type=str,required=True)
+    submit_parser = subp.add_parser('submit')
+    json_read_choice = submit_parser.add_mutually_exclusive_group()
+    json_read_choice.add_argument('-json-file-key',
+                                  default=[], nargs=2)
+    json_read_choice.add_argument('-full-json',
+                                  type=str,
+                                  help="full path to json of splits")
+    submit_parser.add_argument('-user', default='conda-team')
+    submit_parser.add_argument('-queue', default='build_recipes')
+    submit_parser.add_argument('-dry', action='store_true')
+    submit_parser.add_argument('-platforms', required=True,
+                               help="osx-64, linux-64 and/or win-64",
+                               default=[],
```

Comment:
I think the default should be to build all platforms?  Not an important detail now, just trying to think of how this will eventually work.